SelmSample/HistoryView.swift

```diff
@@ -22,13 +22,19 @@ struct HistoryView : View, SelmView {
                 }.onDelete(perform: dispatch • Msg.remove)
             }
         }
+        .onAppear {
+            self.store.subscribe()
```

Comment:
Do we do this to prevent the view from updating when its not on screen?

Reply:
@kylelol Yes.

Reply:
👍  I wonder if we can try to lift that functionality into the library, so we don't have to remember to do this all the time. Thoughts?

Reply:
@kylelol 
Do you have a good idea?

Reply:
I'll create a PR with something, and we can discuss it there 👍

Reply:
@kylelol Thank you!